From 672ad77e95a97da2ee4292911ef158e8e58daff7 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Wed, 10 Oct 2018 18:26:40 +0200
Subject: [PATCH] Eleven: Fix crash after receiving media button intent

* Since eleven is not allowed to launch background services
  we need to use ctx.startForegroundService().

Change-Id: Id8eea0f43c3fec982dc14ee289229793ab908739
---
 src/org/lineageos/eleven/MediaButtonIntentReceiver.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/org/lineageos/eleven/MediaButtonIntentReceiver.java b/src/org/lineageos/eleven/MediaButtonIntentReceiver.java
index 4933825..bd1a55a 100644
--- a/src/org/lineageos/eleven/MediaButtonIntentReceiver.java
+++ b/src/org/lineageos/eleven/MediaButtonIntentReceiver.java
@@ -76,6 +76,6 @@ public class MediaButtonIntentReceiver extends WakefulBroadcastReceiver {
         i.putExtra(MusicPlaybackService.CMDNAME, command);
         i.putExtra(MusicPlaybackService.FROM_MEDIA_BUTTON, true);
         i.putExtra(MusicPlaybackService.TIMESTAMP, timestamp);
-        startWakefulService(context, i);
+        context.startForegroundService(i);
     }
 }
-- 
2.17.1

